source/_components/tts.microsoft.markdown

```diff
@@ -24,18 +24,49 @@ tts:
     api_key: [YourAPIKey]
 ```
 
-Configuration variables:
+{% configuration %}
+api_key:
+  description: Your API key.
+  required: true
+  type: string
+language:
+  description: The language to use. Accepted values are listed in the documentation mentioned below. Note that if you set the language to anything other than the default, you will need to specify a matching voice type as well.
+  required: false
+  default: "`en-us`"
+  type: string
+gender:
+  description: The gender you would like to use for the voice. Accepted values are `Female` and `Male`.
+  required: false
+  default: "`Female`"
+  type: string
+type:
+  description: "The voice type you want to use. Accepted values are listed as the service name mapping [in the documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/Speech/api-reference-rest/bingvoiceoutput)."
+  required: false
+  default: "`ZiraRUS`"
+  type: string
+rate:
+  description: "Change the rate of speaking in percentage. Example values: `25`, `50`."
+  required: false
+  default: 0
+  type: integer
+volume:
+  description: "Change the volume of the output in percentage. Example values: `-20`, `70`."
+  required: false
+  default: 0
+  type: integer
+pitch:
+  description: "Change the pitch of the output. Example values: `high`."
+  required: false
+  default: "`default`"
+  type: string
+contour:
+  description: "Change the contour of the output in percentages. This overrides the pitch setting. See the [W3 SSML specification](http://www.w3.org/TR/speech-synthesis/#pitch_contour) for what it does. Example value: `(0,0) (100,100)`."
+  required: false
+  default: ""
```

Comment:
That is the same as none? Maybe consider removing the default in this case.

Reply:
[See this](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/tts/microsoft.py#L47)

Maybe remove it then?